configs/patternfly.json

```diff
@@ -13,30 +13,317 @@
   ],
   "selectors": {
     "lvl0": {
-      "selector": "header nav [aria-current]",
+      "selector": "header nav li a",
       "global": true,
       "default_value": "Documentation"
     },
     "lvl1": {
-      "selector": "#page-sidebar [aria-expanded='true']",
+      "selector": "#page-sidebar li a",
       "global": true,
       "default_value": "Chapter"
     },
-    "lvl2": "main h1",
-    "lvl3": "main h2",
-    "lvl4": "main h3",
-    "lvl5": "main h4",
-    "lvl6": "main h5",
+    "lvl2": {
+      "selector": "#page-sidebar section li a",
+      "global": true,
+      "default_value": "Chapter"
+    },
+    "lvl3": "main h1",
+    "lvl4": "main h2",
+    "lvl5": "main h3",
+    "lvl6": "main h4",
+    "lvl7": "main h5",
     "text": "main p, main li, main td"
   },
   "selectors_exclude": [
+    ".pf4-l-footer",
     "[class*='live-preview']"
   ],
-  "custom_settings": {
-    "separatorsToIndex": "_"
-  },
   "conversation_id": [
     "772189694"
   ],
-  "nb_hits": 15754
+  "nb_hits": 15754,
+  "custom_settings": {
+    "separatorsToIndex": "_",
+    "synonyms": [
```

Comment:
You can only have up to 20 synonyms if you need more details, you can [have a look here](https://community.algolia.com/docsearch/config-file.html#custom_settings-optional)

Reply:
20 synonym arrays or each of the arrays is limited to 20 synonyms? Alternatively I can set algoliaOptions.synonyms with docsearch from the client?

Reply:
The array of synonyms must be inferior or equals to 20. You can not edit synonym at the search time. Only at the indexing one.